ingestion_server/ingestion_server/api.py

```diff
@@ -163,6 +200,12 @@ def on_post(self, req, res):
                 "task_id": task_id,
                 "status_check": status_url,
             }
+        elif is_bad_request.value:  # set to 1 for bad tasks
```

Comment:
Should we just compare it directly against 1 instead of having to have an explanatory comment?

```suggestion
        elif is_bad_request.value == 1:
```